restic/internal/find_test.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+from unittest import mock
+
+import restic
+from restic.internal import snapshots
+
+
+class FindTest(unittest.TestCase):
+
+    @mock.patch.object(snapshots.command_executor, 'execute')
+    def test_find_simple(self, mock_execute):
+        mock_execute.return_value = '{}'
+
+        restic.find('filename')
+
+        mock_execute.assert_called_with(
+            ['restic', '--json', 'find', 'filename'])
```

Comment:
Can we replace `filename` with `dummyfile`? When the filename is `filename`, it makes it hard for the reader to understand if we're referring to the name of the field or if it's just a file whose name is `filename`. If it's `dummyfile`, it's more obvious that it's the name of the file rather than the field.